Fix login title position

dudchakk opened the pull request:
Added indents so that the sentence "Зареєструватися як студент" now splits into two lines like this:
![Screenshot 2024-07-23 230216](https://github.com/user-attachments/assets/8f3b7edc-f5d1-4310-9309-228760157cd1)

docia commented:
please note that you need at least 3 approvals to merge your PR, so one reviewer is not enough. usually, we add all our teammates, so they can check your work.

dudchakk commented:
> please note that you need at least 3 approvals to merge your PR, so one reviewer is not enough. usually, we add all our teammates, so they can check your work. PS. I've already added myself as a reviewer.

added a few more, thanks for your comment!

however, I was guided by this document:
- 1 reviewer: in case if you have small bug fix, less than 50 rows of code
- 3 reviewers: functional of medium complexity
- 1 expert and 2 devs from team:- in case when we implement complex integrated functionality

should I always add 3+ reviewers then?
